.github/workflows/dockerd.yml

```diff
@@ -48,11 +48,18 @@ jobs:
           version: ${{ env.BUILDX_VERSION }}
           driver-opts: image=${{ env.REPO_SLUG_ORIGIN }}
           buildkitd-flags: --debug
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          tag-sha: true # add git short SHA as Docker tag
       -
         name: Build
         if: steps.build.outputs.result == 'true'
         uses: docker/build-push-action@v3
         with:
+          labels: ${{ steps.meta.outputs.labels }}
```

Comment:
This step only produces BuildKit artifacts, not an image.

Reply:
I have misunderstood the CI implementation, sorry!

Where would I make the change and have it actually apply then?

Reply:
I've updated this PR so it adds the annotations in the right place